### DISCUSSION TITLE
Related videos screen / carousel

### DISCUSSION THREAD

**Discussion by kerem-nextgensoft** [muxinc/elements#861] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Which documentation could use some work?

mux-player

### Is this a discrepancy or an improvement?

improvement

### 🖊 Describe what you'd like to see change in the documentation

Hello,

I'd like to customize the full screen view by rendering some html with links and/or preview of some other video similar to how YouTube does it. How can I accomplish that with mux player?

Additionally, does mux player support local blobs in mp4 format? I have blobs in IndexedDB and I want to use mux player to play those?

### 💡 Describe how you think this change will help the documentation

More people will be able to find out the advanced capabilities of mux player.

**Comment by luwes** [comment]
blob sources should work with the unofficial `src` attribute.
